src/MensaWidget/MWMenuBarTest.class.st

```diff
@@ -61,10 +72,28 @@ MWMenuBarTest >> getAllMeals [
 	meals := OrderedCollection new.
 	self mensaWidget contentMorph allMorphsDo: [:each | 
 		each class = MWMeal ifTrue: 
-			[meals add: (each submorphs at: 1) contents string]].
+			[meals add: (self getMeal: each)]].
 	^ meals
 ]
 
+{
+	#category : #helpers,
+	#'squeak_changestamp' : 'LC 7/8/2024 22:30'
+}
+MWMenuBarTest >> getCanteenName: aMorph [
+	
+	^ aMorph submorphs first submorphs second contents string
+]
+
+{
+	#category : #helpers,
+	#'squeak_changestamp' : 'LC 7/8/2024 22:30'
+}
+MWMenuBarTest >> getMeal: aMorph [
+	
+	^ (aMorph submorphs at: 1) contents string
```

Comment:
You can use `first` instead of `at: 1`

Reply:
same as above we don't need to change this since the morph refactoring implements extra variables for name which I can use then